<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Labo Joly | Logiciels</title>

    <!-- Favicon -->
    <link rel="shortcut icon" type="image/x-icon" href="../carya.ico?">

    <!-- Bootstrap -->
    <link href="../css/bootstrap.min.css" rel="stylesheet">

    <!-- Font awesome -->
    <link rel="stylesheet" href="../css/font-awesome.min.css">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

    <!-- Google analytics -->
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-YQDT7C4E77"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-YQDT7C4E77');
    </script>
  </head>

  <body>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <!-- Brand and toggle get grouped for better mobile display -->
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="index.html">Labo Joly</a>
        </div>

        <!-- Collect the nav links, forms, and other content for toggling -->
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
          <ul class="nav navbar-nav">
            <li><a href="index.html">Accueil <span class="sr-only">(current)</span></a></li>
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Recherche <span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                <li><a href="reproduction.html">Évolution des mécanismes de reproduction</a></li>
                <li><a href="hybridization.html">Spéciation et hybridation</a></li>
                <li class="divider"></li>
                <li><a href="transcriptomics.html">Transcriptomics</a></li>
              </ul>
            </li>
            <li><a href="people.html">Membres</a></li>
            <li class="active"><a href="softwares.html">Logiciels</a></li>
            <li><a href="publications.html">Publications</a></li>
            <li><a href="journal_club.html">Journal Club</a></li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="../en/softwares.html">English</a></li>
          </ul>
        </div><!-- /.navbar-collapse -->
      </div><!-- /.container -->
    </nav>

    <img class="media-object" src="../images/Population_Vimont.jpg" alt="..." width="100%">

    <div class="container">
      <p></p>
    </div>

    <div class="container">
      <div class="jumbotron">
        <h2>Softwares</h2>
        <p>Most softwares and scripts are available on <a href="http://www.github.com/simjoly" target="_blank"><i class="fa fa-github"></i> github</a>. The most important softwares are briefly described below...</p>
      </div>
    </div>

    <div class="container">
      <div class="row">

        <div class="col-xs-12 col-md-6"> 
          <div class="panel panel-default">
            <div class="panel-heading">
              <h3 class="panel-title">JML</h3>
            </div>
            <div class="panel-body">
              <h1>JML</h1>
              <h4>Testing hybridization from species trees</h3>
              <p>© Simon Joly, 2011-2013</br><em>Montreal Botanical Garden</em></p>
              <h3>Introduction</font></h3>
                <p>JML is a program that implements the posterior predictive checking method of Joly et al. 2009 for detecting hybridization events. Replicate datasets are simulated from the posterior distribution of species trees (with branch lengths and population sizes) using the coalescent. Then a test quantity, the minimum distance between sequences of two species, is calculated for all replicated datsets to generate a posterior predictive distribution. The observed distance is finally compared to this distribution to calculate the probability that the observed distance is caused by hybridization.</p>
              <h3>Download and Installation</h3>
                <p>Source code and binaries for the program are available on Github</p>
                <p><a href="https://github.com/simjoly/jml/releases" target="_blank"><i class="fa fa-github fa-lg"></i> JML</a></p>
              <h3>Citation</h3>
              <p>Joly, S. 2012. JML: Testing hybridization from species trees. <em>Molecular Ecology Ressources</em>. 12:179-184</p>
              <p>Joly, S., P.A. McLenachan, P.J. Lockhart. 2009. A statistical approach for distinguishing hybridization and incomplete lineage sorting. <em>The American Naturalist</em> 174: e54-e70</p>
            </div>
          </div> <!-- panel-default -->
        </div> <!-- col-xs-12 col-md-6 -->

        <div class="col-xs-12 col-md-6">
          <div class="panel panel-default">
            <div class="panel-heading">
              <h3 class="panel-title">POFAD</h3>
            </div>
            <div class="panel-body">
              <h1>POFAD</h3>
              <h4>Phylogeny of Organisms From Allelic Data</h4>
              <p>© Simon Joly, 2006-2014<br/>
                <em>Montreal Botanical Garden</em></p>
             <h3>Introduction</font></h3>
              <p>POFAD implements several approaches for estimating genetic distances between individuals (organisms) in the presence of intra-individual polymorphisms or SNPs due to allelic variation, polyploidy, gene families, etc. Read the manual for further information.</p>
              <h3>Download and Installation</h3>
                <p>Source code and binaries for the program are available on Github</p>
                <p><a href="https://github.com/simjoly/pofad/releases" target="_blank"><i class="fa fa-github fa-lg"></i> POFAD</a></p>
              <h3>References</h3>
              <p>Joly, S. and Bruneau. 2006. Incorporating allelic variation for reconstructing the evolutionary history of organisms from multiple genes: an example from <i>Rosa</i> in North America. <i>Systematic Biology</i>. 55(4):623-636.</p>
              <p>Joly, S. D. Bryant, and P. J. Lockhart. 2015. Flexible methods for estimating genetic distances from nucleotide data. <em>Methods in Ecology and Evolution</em>. 6(8):938-948</p>
            </div>
          </div> <!-- panel-default -->

        </div> <!-- col-xs-12 col-md-6 -->
      </div> <!-- row  -->


      <!-- FOOTER -->
      <footer>
        <hr>
        <p class="pull-right"><a href="#">Retour en haut</a></p>
        <p>Site hébergé sur <a href="https://github.com/plantevolution/plantevolution.github.io" target="_blank"><i class="fa fa-github fa-lg"></i> GitHub</a></br>
          Construit avec <a href="getbootstrap.com">Bootstrap</a> &middot; <a href="fortawesome.github.io/Font-Awesome/">Font awesome</a><br>
          &copy; 2015 Simon Joly &middot; <a href="#">Termes</a></p>
      </footer>
    </div><!-- /.container -->


    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="../js/bootstrap.min.js"></script>
  </body>
</html>